src/main/java/pages/LoginPage.java

```diff
@@ -0,0 +1,30 @@
+package pages;
+
+import base.PageBase;
+import org.openqa.selenium.By;
+
+public class LoginPage extends PageBase {
+
+    private static final By LOGIN_FIELD = By.id("mailbox:login");
+    private static final By PASSWORD_FIELD = By.id("mailbox:password");
+    private static final By SUBMIT_BUTTON = By.id("mailbox:submit");
+
+    public MainPage populateLoginPage(String login, String password) {
```

Comment:
Rename into "login". I expected that method named "populateLoginPage" will only set username and password, but not to click submit - bad name